cornflow/tests/custom_test_case.py

```diff
@@ -31,6 +31,8 @@ def setUp(self):
         self.url = None
         self.model = None
         self.payload = None
```

Comment:
payload is not needed as a property. It's always read and used in the same function.

Reply:
Done